scripts/lint_repo.py

```diff
@@ -0,0 +1,97 @@
+# This script will check that the directory
```

Comment:
This should go directory to planemo, I think

Reply:
You mean directly?

Reply:
Yeah, `workflow_lint` should do this.